engine/execution/execution_test.go

```diff
@@ -57,7 +59,7 @@ func TestExecutionFlow(t *testing.T) {
 		unittest.WithKeys,
 	)
 
-	identities := unittest.CompleteIdentitySet(colID, conID, exeID, verID)
+	identities := unittest.CompleteIdentitySet(colID, conID, exeID, verID).Sort(order.Canonical)
```

Comment:
Tests were flakey because the identities were not sorted. 

It's now fixed by sorting it first